packages/volto/src/config/PublicUiRoutes.jsx

```diff
@@ -0,0 +1,17 @@
+import config from '@plone/volto/registry';
+
+// PublicUi routes that are noContentRoutes, and we won't cms-ui
+// You can include either RegEx or a string representing the ending of the
+// nonContentRoute eg. '/add' will match '/foo/bar/add'
+export const publicUiRoutes = [
+  '/login',
+  '/logout',
+  '/sitemap',
+  '/register',
+  '/search',
+  '/change-password',
+  '/contact-form',
+  '/register',
+  /\/passwordreset\/.*$/,
+  '/passwordreset',
```

Comment:
```suggestion
  '/login$',
  '/logout$',
  '/sitemap$',
  '/register$',
  '/search$',
  '/change-password$',
  '/contact-form$',
  /\/passwordreset($|\/)/,
```
make regexp more rigingorous to solve the problem of match something like `/searching-content` ...

Reply:
with $ this means that you don't consider parameters.. i've just copied path from nonContentRoutes. 
I think for now they colud be remain the same

Reply:
`$` would not match `/login/`, for example, which would be bad assuming they were redirected from a page that requires authentication and want to redirect them back to that page after authentication. I think it is better to leave all these open-ended here.

I'm not sure why we have this one-off, though. Is there a good reason not to simplify it to just the string `/passwordreset`?

```js
/\/passwordreset($|\/)/
```

Reply:
I've doublchecked code here. IMO, thinking of fixing the regexes here is useless. The real regexes used are in `nonContentRoutes`, the `publicNonContentRoutes` list is about the regexes in `nonContentRoutes` that should not be used for matching when it comes to evaluating `isCMSui`. For better readability, I moved the two lists in the same file and added an explicit comment.

Reply:
This makes sense to me.